tests/sv-comp/gen/gen.ml

```diff
@@ -0,0 +1,33 @@
+let generate_rule c_dir_file =
+  let dir = Filename.dirname c_dir_file in
+  let c_file = Filename.basename c_dir_file in
+  let file = Filename.chop_extension c_file in
+  Printf.printf {|
+(subdir %s
+ (rule
+  (deps
+   (sandbox always) ; must sandbox to prevent arg.dot-s from parallel runs from interfering
+   (package goblint) ; depend on entire goblint package for svcomp21 conf
+   (:c %s)
+   (:prop %%{project_root}/tests/sv-comp/unreach-call-__VERIFIER_error.prp))
+  (target %s.output)
+  (enabled_if %%{bin-available:graph-easy})
+  (action
+   (progn
+    (ignore-outputs
+     (run goblint --conf svcomp21.json --set ana.specification %%{prop} %%{c} --enable witness.graphml.uncil --enable exp.argdot --enable exp.arg --set exp.argdotlabel empty))
```

Comment:
Could you elaborate why you use the `svcomp21` config for these? Would it not make sense to use a more recent config (maybe the most recent?) in order to catch bugs that only surface with new analyses / our new config?

Reply:
I just used `svcomp21` because that's what the old witness generation scripts still reference and I had old witnesses for. I now changed it to `svcomp` (without autotuning to keep things somewhat under control).
This does mean that as we change our SV-COMP setup, these ARG tests might fail only due to precision changes. But they're rather simple, so this should rarely happen.